### ISSUE TITLE
Decoupling version numbers of different specifications

### ISSUE THREAD

**Issue by constantinpape** [ome/ngff#47]
While working on #46 I noticed that we now need to bump the version number for the HCS spec, although nothing in it has changed. In the future with more specifications for different OME-NGFF formats, this can become pretty confusing, because any change in one of the specifications will require a "global" version bump.

I am not sure if there is a better solution, because having diverging version numbering in the same repository could introduce it's own set of problems. But I still wanted to bring this up for discussion.

**Comment by joshmoore** [comment]
Agreed. My (strong) preference would be to introduce a single top-level item that provides the version:

```
{
    "@context": "https://ngff.openmicroscopy.org/contexts/0.2"
}
```

that would allow for a json-ld parsing: https://json-ld.org/spec/latest/json-ld/#the-context

**Comment by constantinpape** [comment]
Closed in favor of #83
